server/server.go

```diff
@@ -111,6 +117,16 @@ func (server HTTPServer) readOrganizationID(writer http.ResponseWriter, request
 		return 0, errors.New(message)
 	}
 
+	if identityContext != nil && Environment == "production" {
+		identity := identityContext.(Identity)
+		if Environment == "production" && identity.Internal.OrgId != organizationIDParam {
```

Comment:
enum type there?

Reply:
or better: please add "production bool" into Configuration and change this line to something like: if server.config.production && ...